docs/dashboards/visualization/visualization.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Isn't this file to be renamed as index.md?

Reply:
Right. We can keep index.md as synonym for entry point. My bad